DockerStatic: Hostname Missing From SLES12 container

JDK17 s390x_linux triage

Machine related failures: Cannot run program "hostname": error=2, No such file or directory on test-docker-sles12-s390x-1
```
java.lang.RuntimeException: Get hostname failed.
	at PostThruProxyWithAuth.getHostname(PostThruProxyWithAuth.java:263)
	at PostThruProxyWithAuth.doClientSide(PostThruProxyWithAuth.java:180)
	at PostThruProxyWithAuth.main(PostThruProxyWithAuth.java:126)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.sun.javatest.regtest.agent.MainWrapper$MainTask.run(MainWrapper.java:138)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: Cannot run program "hostname": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at jdk.test.lib.process.ProcessTools.privilegedStart(ProcessTools.java:611)
	at jdk.test.lib.process.ProcessTools.executeProcess(ProcessTools.java:442)
	at jdk.test.lib.process.ProcessTools.executeProcess(ProcessTools.java:421)
	at jdk.test.lib.process.ProcessTools.executeProcess(ProcessTools.java:408)
	at jdk.test.lib.process.ProcessTools.executeProcess(ProcessTools.java:489)
	at jdk.test.lib.process.ProcessTools.executeCommand(ProcessTools.java:537)
	at PostThruProxyWithAuth.getHostname(PostThruProxyWithAuth.java:260)
	... 8 more
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:314)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:244)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 17 more
```

_Originally posted by @smlambert in https://github.com/adoptium/aqa-tests/issues/5137#issuecomment-1998610945_

--- a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12
+++ b/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12
@@ -2,7 +2,7 @@ FROM registry.suse.com/suse/sles12sp5:latest
 
 RUN zypper ar https://download.opensuse.org/distribution/leap/15.4/repo/oss/ sles15oss
 RUN zypper --gpg-auto-import-keys refresh
-RUN zypper update -y && zypper install -y wget perl openssh-server unzip zip tar gzip
+RUN zypper update -y && zypper install -y wget perl openssh-server unzip zip tar gzip hostname
 
 RUN wget 'https://api.adoptium.net/v3/binary/latest/17/ga/linux/x64/jdk/hotspot/normal/eclipse?project=jdk' -O /tmp/jdk17.tar.gz
 RUN mkdir -p /usr/lib/jvm/jdk17 && tar -xpzf /tmp/jdk17.tar.gz -C /usr/lib/jvm/jdk17 --strip-components=1

